server/monitor_test.go

```diff
@@ -4050,7 +4050,9 @@ func TestMonitorAccountz(t *testing.T) {
 
 	body = string(readBody(t, fmt.Sprintf("http://127.0.0.1:%d%s?unused=1", s.MonitorAddr().Port, AccountStatzPath)))
 	require_Contains(t, body, `"acc": "$G"`)
+	require_Contains(t, body, `"acc_name": "$G"`)
```

Comment:
Be good to have a test with JWTs that show they are different.

Reply:
Will add.